src/wiki/installing/windows.md

```diff
@@ -30,15 +30,15 @@ If you encounter any problems, **don't worry**, we have a guide that goes over s
 Please click [here](../../getting-started/installing-polymc) for more.
 
 
-### A Note on Legacy Windows Support (Windows 7)
+### A Note on Legacy Windows Support (Windows 7, 8)
 
 
-While it may be **possible** to install PolyMC on this legacy operating system, it is **not recommended, nor is it officially supported**. We may or may not choose to provide support for any issues you encounter.
+While it may be **possible** to install PolyMC on these legacy operating systems, it is **not recommended, nor is it officially supported**. We may or may not choose to provide support for any issues you encounter.
 
-While PolyMC currently works on Windows 7, support could break at any time.
+PolyMC does currently work on Windows 7, however, support could break at any time.
 
-If you want a more technical explanation, it's because msys2 and Cygwin will start targeting 8.1, that we use for building PolyMC on Windows ([see this](https://www.msys2.org/docs/windows_support/))
+If you want a more technical explanation, it's because msys2 and Cygwin dependancies, that we use for building PolyMC on Windows, will start targeting 8.1  ([see this](https://www.msys2.org/docs/windows_support/)).
```

Comment:
```suggestion
If you want a more technical explanation, it's because msys2 and Cygwin, that we use for building PolyMC on Windows, will start targeting 8.1 ([see this](https://www.msys2.org/docs/windows_support/)).
```

Reply:
yeah I wasn't too sure about that one, thank you